mthree/mitigation.py

```diff
@@ -155,12 +155,11 @@ def cals_from_system(self, qubits=None, shots=None, method='balanced',
             qubits = range(self.num_qubits)
         self.cal_method = method
         self.rep_delay = rep_delay
+        self.cals_file = cals_file
```

Comment:
`self.cals_file` is not defined inside of the `__init__` function, which is causing the linter to raise an error.  You should do something like `self._cals_file = None`, or the like in the `__init__`.  The underscore is to highlight that it is not meant to be user facing